### ISSUE TITLE
Update compilation instructions

### ISSUE THREAD

**Issue by FeedbackReporter** [tchegito/zildo#184]
The only instructions I could find are here:
https://github.com/tchegito/zildo/blob/wiki/SetUpZildo.md

Try as I may, I am not able to compile with Android Studio or Eclipse for any platform using Win10 or Linux Mint 20.
Libraries seem to be pretty out of date.

**Comment by tchegito** [comment]
Yes you're right. I had a bad time updating all the libs (gradle, new eclipse, Android Studio ...).
Now that's ok.
